lib/1.4/dml-builtins.dml

```diff
@@ -1386,6 +1575,10 @@ The `init_as_subobj` inherits the [`init`](#init) and
 template init_as_subobj is (connect, init) {
     param classname : const char *;
     param configuration default "none";
+    #if (dev.thread_aware) {
```

Comment:
TODO: document `init_as_subobj` as incompatible with thread-aware device models.

Reply:
This is because there is no way to prevent the created subobject from being placed in the same, independent thread domain as the device object; which is wrong if the subobject device class is _not_ thread aware. (Well, ok, there is _technically_ one but it is horribly hacky.) The Framework team knows about this issue.